src/server.ts

```diff
@@ -0,0 +1,16 @@
+// const Koa = require('koa');
```

Comment:
I think you can remove this commented-out line

Reply:
oops ps